### DISCUSSION TITLE
Laravel-backpack show total of column

### DISCUSSION THREAD

**Discussion by Thomanu** [Laravel-Backpack/community-forum#253] [Q&A (Help)] [answered]
<img width="995" alt="Capture d’écran 2022-06-10 à 12 00 31" src="https://user-images.githubusercontent.com/72254639/173048368-96b69946-2b27-40f3-9c84-21605c019603.png">
Is there a way to calculate all the durations and display them at the bottom using backpack? Thanks !

**Comment by christmex** [comment]
you can create a custom function inside your **setupListOperation** or maybe create **Accessor** inside your model please provide more details so maybe I can help you with the code

Thanks

**Reply by tabacitu** [reply, marked as answer]
Hmm... I don't think that would work. I think what they were trying to achieve (6 months ago, mind you 👀😔) was a TOTAL, placed after the table. In the past, I've only done something like that by [creating a custom Widget](https://backpackforlaravel.com/docs/5.x/base-widgets#creating-a-custom-widget-type-1).

**Reply by christmex** [reply]
Ohh, I'm sorry, maybe I miss understand, and I don't see that was 6 months ago 😔

**Reply by tabacitu** [reply]
Oh yeah, that too 😀 You probably got a notification because I moved all Discussions from the CRUD repo to the `ideas` repo, and renamed `ideas` to `community-forum`. The plan is to have all conversations in one spot, until it we turn them into tasks.
